### PR TITLE
Add unit test for OperandBindInfo

### LINKED ISSUE #295: Adding unit test for OperandBindInfo
/kind feature

**Describe the solution you'd like**
[A clear and concise description of what you want to happen.]

Add unit test to cover the OperandBindInfo controller.

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

### PR DESCRIPTION
**Pull request by horis233**
**What this PR does / why we need it**:

Add unit test for OperandBindInfo

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #295 

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/controller/operandbindinfo/operandbindinfo_controller.go [hunk 1]
```diff
@@ -139,17 +140,31 @@ func (r *ReconcileOperandBindInfo) Reconcile(request reconcile.Request) (reconci
 	if len(requestNamespaces) == 0 {
 		return reconcile.Result{}, nil
 	}
+	// Get the operand namespace
+	var operandNamespace string
+	for _, op := range registryInstance.Spec.Operators {
+		if op.Name == bindInfoInstance.Spec.Operand {
+			operandNamespace = op.Namespace
```
Comment (DanielXLee) [inline pkg/controller/operandbindinfo/operandbindinfo_controller.go:147]: Add a `break` is better when the namespace found

Comment (horis233) [reply pkg/controller/operandbindinfo/operandbindinfo_controller.go:147]: Will add

File: pkg/controller/operandbindinfo/operandbindinfo_controller_test.go [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	"github.com/IBM/operand-deployment-lifecycle-manager/pkg/apis/operator/v1alpha1"
+)
+
+// TestRequestController runs ReconcileOperandRequest.Reconcile() against a
+// fake client that tracks a OperandRequest object.
+func TestRequestController(t *testing.T) {
```
Comment (DanielXLee) [inline pkg/controller/operandbindinfo/operandbindinfo_controller_test.go:42]: the func name should be `TestBindInfoController`

Comment (horis233) [reply pkg/controller/operandbindinfo/operandbindinfo_controller_test.go]: Done

File: pkg/controller/operandbindinfo/operandbindinfo_controller.go (hunk 1 above)
Comment (horis233) [reply pkg/controller/operandbindinfo/operandbindinfo_controller.go:147]: @DanielXLee Done

**Pull request merged**